po/ru.po

```diff
@@ -0,0 +1,155 @@
+"POT-Creation-Date: 2019-11-11 21:08+0900\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Max Usmanov <usmanovmax@protonmail.com>\n"
+"Language-Team: \n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: src/Views/CountDownView.vala:48
+msgid "Cancel the countdown"
+msgstr "Отменить обратный отсчёт"
+
+#: src/Views/CountDownView.vala:54 src/Views/CountDownView.vala:125
+#: src/Views/CountDownView.vala:147
+msgid "Pause the countdown"
+msgstr "Остановить обратный отсчёт"
+
+#: src/Views/CountDownView.vala:138
+msgid "Resume the countdown"
+msgstr "Продолжить обратный отсчёт"
+
+#: src/Views/RecordView.vala:62
+msgid "Cancel recording"
+msgstr "Отменить запись"
+
+#: src/Views/RecordView.vala:68
+msgid "Finish recording"
+msgstr "Закончить запись"
+
+#: src/Views/RecordView.vala:76 src/Views/RecordView.vala:98
+#: src/Views/RecordView.vala:129 src/Views/RecordView.vala:141
+msgid "Pause recording"
+msgstr "Остановить запись"
+
+#: src/Views/RecordView.vala:119
+msgid "Resume recording"
+msgstr "Продолжить запись"
+
+#: src/Views/WelcomeView.vala:33
+msgid "Recording"
+msgstr "Записываю"
+
+#: src/Views/WelcomeView.vala:35
+msgid "Delay in seconds:"
+msgstr "Задержка в секундах:"
+
+#: src/Views/WelcomeView.vala:41
+msgid "Length in seconds:"
+msgstr "Длина в секундах:"
+
+#: src/Views/WelcomeView.vala:47
+msgid "Record from:"
+msgstr "Запись из:"
+
+#: src/Views/WelcomeView.vala:51
+msgid "Microphone"
+msgstr "Микрофон"
+
+#: src/Views/WelcomeView.vala:52
+msgid "Computer"
+msgstr "Компьютер"
+
+#: src/Views/WelcomeView.vala:53
+msgid "Both"
+msgstr "Оба"
+
+#: src/Views/WelcomeView.vala:56
+msgid "Saving"
+msgstr "Сохраняю"
+
+#: src/Views/WelcomeView.vala:58
+msgid "Format:"
+msgstr "Формат:"
+
+#: src/Views/WelcomeView.vala:63
+msgid "AAC"
+msgstr "ААС"
+
+#: src/Views/WelcomeView.vala:64
+msgid "FLAC"
+msgstr "FLAC"
+
+#: src/Views/WelcomeView.vala:65
+msgid "MP3"
+msgstr "MP3"
+
+#: src/Views/WelcomeView.vala:66
+msgid "Ogg Vorbis"
+msgstr "Ogg Vorbis"
+
+#: src/Views/WelcomeView.vala:67
+msgid "Opus"
+msgstr "Opus"
+
+#: src/Views/WelcomeView.vala:68
+msgid "WAV"
+msgstr "WAV"
+
+#: src/Views/WelcomeView.vala:71
+msgid "Automatically save files:"
+msgstr "Автоматически сохранять файлы:"
+
+#: src/Views/WelcomeView.vala:79
+msgid "Choose a default destination"
+msgstr "Выберите место по умолчанию"
+
+#: src/Views/WelcomeView.vala:105
+msgid "Start recording"
+msgstr "Начать запись"
+
+#. TRANSLATORS: The name of the folder which recordings are saved
+#: src/Views/WelcomeView.vala:139
+msgid "Recordings"
+msgstr "Записи"
+
+#: src/MainWindow.vala:80
+msgid "Unable to Create an Audio File"
+msgstr "Невозможно Создать Аудиофайл"
+
+#: src/MainWindow.vala:81
+msgid ""
+"A GStreamer error happened while recording, the following error message may "
+"be helpful:"
+msgstr ""
+"Произошла ошибка "GStreamer" во время записи, следующее сообщение ошибки "
```

Comment:
```suggestion
"Произошла ошибка \"GStreamer\" во время записи, следующее сообщение ошибки "
```

If you want to use quotation marks in your translations, you need to escape it with `\`.